website/content/en/docs/cli/operator-sdk_generate_bundle.md

```diff
@@ -30,62 +38,55 @@ operator-sdk generate bundle [flags]
 
 ```
 
-  # Generate bundle files and build your bundle image with these 'make' recipes:
-  $ make bundle
-  $ export USERNAME=<quay-namespace>
-  $ export BUNDLE_IMG=quay.io/$USERNAME/memcached-operator-bundle:v0.0.1
-  $ make bundle-build BUNDLE_IMG=$BUNDLE_IMG
-
-  # The above recipe runs the following commands manually. First it creates bundle
-  # manifests, metadata, and a bundle.Dockerfile:
-  $ make manifests
-  /home/user/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-  $ operator-sdk generate kustomize manifests
-
-  Display name for the operator (required):
-  > memcached-operator
-  ...
-
+  # If running within a project, make sure a 'config' kustomize directory exists and has a 'config/manifests':
```

Comment:
What about we use exactly the same wording all? See https://github.com/operator-framework/operator-sdk/pull/4514/files#r577182711

Reply:
Done